pkg/hoist/hoist_launchable.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/square/p2/pkg/auth"
 	"github.com/square/p2/pkg/cgroups"
 	"github.com/square/p2/pkg/launch"
+	"github.com/square/p2/pkg/manifest"
```

Comment:
I don't think launchables shouldn't know about pod manifests per previous discussions around versioning of manifests and separation of concerns.